app/src/main/java/xyz/block/gosling/features/agent/Tools.kt

```diff
@@ -497,6 +498,15 @@ object ToolHandler {
         }
     }
 
+    @Tool(
+        name = "pressEnter",
+        description = "Press this after entering text into a search field if there is no button to take action.",
+    )
+    fun pressEnter(args: JSONObject): String {
+        Runtime.getRuntime().exec(arrayOf("input", "keyevent", "KEYCODE_ENTER"))
+        return "Pressed enter"
+    }
```

Comment:
Nice! This looks good to me! We could also have a tool that can find submit/done buttons on the keyboard as well. I think `Enter` has to be implemented by the developer, which most should by default. And I'm guessing this happens on UIs where we don't have the keyboard visible with its own `Done/Search/Next` button